telos-frontend/src/components/steam/steam.jsx

```diff
@@ -0,0 +1,63 @@
+import { DataGrid } from '@material-ui/data-grid';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+// Add in your steam name
+// More information on: https://github.com/UOA-SE701-Group3-2021/3Lancers/pull/287
+const steamName = 'SteamName';
+
+const steamColumns = [
+  { field: 'name', headerName: 'Game', width: 200 },
+  { field: 'playtime_2weeks', headerName: 'Recently played (Min)', width: 200 },
+  { field: 'playtime_forever', headerName: 'Total played (Min)', width: 170 },
+];
+
+const useStyles = makeStyles({
```

Comment:
Is it possible for this styling to be moved into a CSS file?

Reply:
I did a quick change to css file but had no success in it working for material UI.  Feel free to change anything in the file.